pkg/provider/aws/knowledgebase/api_gateway.go

```diff
@@ -212,5 +207,5 @@ func createRoutesForIntegration(appName string, routes []core.Route, refs core.A
 		}
 		dag.AddDependenciesReflect(integration)
 	}
-	return nil
+	return merr
```

Comment:
needs to be `merr.ErrOrNil()`

Reply:
well i didnt use multierr i used go errors and did errors.Join so this should just be the error, i just didnt want to name it err because then i didnt know what to name the local errors

Reply:
I didn't catch the `errors.Join` - this'll be the first we use that in the code base. 👍